pkg/cableengine/cableengine.go

```diff
@@ -156,13 +157,14 @@ func (i *engine) installCableWithNATInfo(rnat *natdiscovery.NATEndpointInfo) err
 	i.Lock()
 	defer i.Unlock()
 
-	if _, ok := i.natDiscoveryPending[rnat.Endpoint.Spec.CableName]; !ok {
+	familyCableName := rnat.Endpoint.Spec.GetFamilyCableName(rnat.Family)
+	if _, ok := i.natDiscoveryPending[familyCableName]; !ok {
 		return nil
 	}
 
-	i.natDiscoveryPending[rnat.Endpoint.Spec.CableName]--
-	if i.natDiscoveryPending[rnat.Endpoint.Spec.CableName] == 0 {
-		delete(i.natDiscoveryPending, rnat.Endpoint.Spec.CableName)
+	i.natDiscoveryPending[familyCableName]--
+	if i.natDiscoveryPending[familyCableName] == 0 {
+		delete(i.natDiscoveryPending, familyCableName)
```

Comment:
I think `installedCables` below will have to index by `familyCableName` as well but we can address that when cable drivers handle the family.